service/awsconfig/v8/resource/cloudformation/delete.go

```diff
@@ -11,38 +11,62 @@ import (
 )
 
 func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange interface{}) error {
-	cluster, err := key.ToCustomObject(obj)
+	customObject, err := key.ToCustomObject(obj)
 	if err != nil {
 		return microerror.Mask(err)
 	}
-
-	deleteStackInput := awscloudformation.DeleteStackInput{
-		StackName: aws.String(key.MainGuestStackName(cluster)),
-	}
-	_, err = r.clients.CloudFormation.DeleteStack(&deleteStackInput)
+	stackInputToDelete, err := toStackState(deleteChange)
```

Comment:
There is the `StackState` type dispatched. IMO we should propagate the same state within a resource. This is currently not the case within the `cloudformation` resource.

Reply:
+1 Good to improve this so StackState is always dispatched.